README.md

```diff
@@ -33,3 +21,50 @@ run `./gradlew addKtlintFormatGitPreCommitHook`. As the result, the `.git` folde
 - [x] [Multiple Operator Comparison](./docs/transformations/MultipleOperatorComparison.md)
 - [x] [Multiple Target Assignment](./docs/transformations/MultipleTargetAssignmentTransformation.md)
 - [x] [Outer Not Elimination](./docs/transformations/OuterNotElimination.md)
+
+## Installation
+
+Just clone the repo by `git clone https://github.com/JetBrains-Research/ast-transformations`.
+
+
+## Getting started as a tool
+
+Run the command `./gradlew :cli -Pinput=<Input directory with python files> -Poutput=<Output directory> -Pyaml=<YAML config path>`.
+The YAML configuration file example can be found [here](./config.yaml).
+Set of the transformations is applied `n transformations` times. Each round for each transformation the random  
```

Comment:
Maybe rename `n transformations` parameter? Now it looks like the number of transformations, while it's about the number of times set of the transformations should be applied. How about `n apply` or `n times`?

Reply:
It also looks like we apply transformation's n times consistently, like FIle -> Transformed(File) -> Transformed(Transformed(FIle)) and so on. But according to the implementation, we do the first step (FIle -> Transformed(File)) n times. Is it correct?